### PR TITLE
gui implementation

### PR DESCRIPTION
**Pull request by adsau59**

### CODE REVIEW

File: gui/control.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+async function main(){
+    const errorCode = await initialize();
+    if(errorCode) showToast("Error!", errorToMessage[errorCode]);
+    if(!global_state.flags.parsecDataLocationFound) showToast("Error!", errorToMessage[error.PARSEC_NOT_INSTALLED])
+    if(!global_state.flags.parsecdFound) showToast("Error!", error.PARSECD_NOT_IN_DEFAULT);
+
+    global_state.onConfigChanged.push(render);
+    addAccountButton.addEventListener('click', addButtonPressed);
```
Comment (electr0de) [inline gui/control.js:59]: Maybe this is the code that's causing the delay in rendering in some PC's?

File: gui/elements.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const templates = require("./template");
+
+async function showYesNoPopup(description, negative_text, positive_text){
+    // const id = shortUUID.generate().toString();
+    const html = templates.generate_yes_no_popup(description, negative_text, positive_text);
+    const popupDiv = document.getElementById("popup-div");
+    popupDiv.innerHTML = html;
+    const modal = new bootstrap.Modal(document.querySelector("#yes-no-popup"));
```
Comment (electr0de) [inline gui/elements.js:8]: This is making intelliJ unhappy

**Review by electr0de — COMMENTED** [review]
Everything else seems great, but in the hindsight, using react/angular would have been better

File: gui/elements.js (hunk 2 above)
Comment (adsau59) [reply gui/elements.js:8]: yeah as we are importing it through html and not the node way, can't find a way to fix it other than commenting suppress statement

Comment (adsau59) [reply gui/elements.js:10]: done

File: gui/control.js (hunk 1 above)
Comment (adsau59) [reply gui/control.js:59]: what should i do about it? we need to run initialize and do some check before everything is rendered right?

**Pull request merged**